upf/test/test_upf.py

```diff
@@ -1048,6 +1061,81 @@ def verify_gtp_no_echo(self, to_send):
         self.send_from_grx(to_send)
         self.if_grx.assert_nothing_captured()
 
+    def send_from_sgi_to_ue(self, payload=None, l4proto=UDP, ue_port=12345,
+                                remote_port=23456, remote_ip=None, **kwargs):
+        if remote_ip is None:
+            remote_ip = self.sgi_remote_ip
+        to_send = Ether(src=self.if_sgi.remote_mac,
+                        dst=self.if_sgi.local_mac) / \
+            self.IP(src=remote_ip, dst=self.ue_ip) / \
+            l4proto(sport=remote_port, dport=ue_port, **kwargs)
+        if payload is not None:
+            if isinstance(payload, bytes):
+                payload = Raw(payload)
+            to_send /= payload
+        self.if_sgi.add_stream(to_send)
+        self.pg_enable_capture(self.pg_interfaces)
+        self.pg_start()
+        return
+
+    def send_from_ue_to_sgi(self, payload=None, l4proto=UDP, ue_port=12345,
```

Comment:
This function doesn't appear to send anything. So, rename? (and maybe extract to a PFCPHelper method that should also be used as part of other `send_` functions)